lib/-private/todo-handler.js

```diff
@@ -74,8 +70,8 @@ module.exports = class TodoHandler {
         (result) =>
           result.filePath === todo.filePath &&
           result.rule === todo.ruleId &&
-          result.line === todo.line &&
-          result.column === todo.column
+          result.line === todo.range.start.line &&
+          result.column === todo.range.start.column
```

Comment:
I think this should also be updated to care RE: `source`, right?

Reply:
Hmm, it's almost as if we have to do a secondary match on these...

Reply:
We should discuss this more, but the crux of this issue is that once we identify matches as `stable`, we further need a way to associate the stable items with their associated lint results. This is critical for us to be able to mutate the severity. As it stands now, the new fuzzy matching means that the logic above won't work for fuzzy matches, as we can't associate the result anymore. We effectively need a way to brand the lint result onto the todo data, such that it can be compared when processing stable matches.

Reply:
This has been resolved via https://github.com/ember-template-lint/ember-template-lint-todo-utils/pull/277